src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -381,8 +381,16 @@ private void ProcessTraceroute(string targetNameOrAddress)
 #endif
                 var hopAddressString = discoveryReply.Address.ToString();
 
-                if (!TryResolveNameOrAddress(hopAddressString, out string routerName, out _))
+                try
+                {
+                    if (!TryResolveNameOrAddress(hopAddressString, out string routerName, out _))
```

Comment:
After we rename the method to TrySomeThing() it is expected that the method does not throw.
To fix this we would have to refactor too much so it is LGTM

Reply:
Yeah, I agree. This was meant to be a quick fix for functionality.

We can refactor later to remove the need to throw, I would need to change where errors are generated.